sysutils/rawhide/files/patch-Makefile.diff

```diff
@@ -0,0 +1,17 @@
+Do not overwrite MacPorts's CFLAGS.
+--- Makefile
++++ Makefile
+@@ -135,11 +135,11 @@ RAWHIDE_DEFINES = \
+ #UBSAN_CFLAGS = -fsanitize=undefined
+ #UBSAN_LDFLAGS = -fsanitize=undefined
+ 
+-CC = cc
++#CC = cc
```

Comment:
You can just remove the line. It's simpler, shorter, easier to read than commenting it out. Or, if you use the makefile portgroup, I don't think you need it, since the makefile portgroup overrides select Makefile variables, and `CC` is one of them (see `makefile.override` in makefile-1.0.tcl).

Reply:
That CC line is modified by the configure script when $CC is set. I don't know if that changes your assessment. But I don't think that it needs to be commented out or deleted. At least, I can't see any reason why it should be. It's probably harmless if CC is also set when make is run, because it would have the same value either way, but it's unnecessary to remove it.